cmd/controller-manager/app/controller-manager.go

```diff
@@ -303,7 +303,12 @@ func setOptionsByFederationConfig(opts *options.Options) {
 	if fedConfig == nil {
 		// FederationConfig could not be sourced from --federation-config or from the API.
 		// create a new `FederationConfig` with default values.
-		fedConfig = &corev1a1.FederationConfig{}
+		fedConfig = &corev1a1.FederationConfig{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      util.FederationConfigName,
+				Namespace: util.DefaultFederationSystemNamespace,
```

Comment:
I don't think this is the namespace that we necessarily want; the logic for the namespace should be the same as: https://github.com/kubernetes-sigs/federation-v2/blob/master/cmd/controller-manager/app/controller-manager.go#L115-L121

Reply:
The config _needs_ to be in the federation namespace.  That's what makes it discoverable by `kubefed2`.

Reply:
As discussed, I have changed the logic here to compute the right value of the namespace based on how we're planning to deploy federation.